akka-testkit/src/test/scala/akka/testkit/AkkaSpec.scala

```diff
@@ -23,8 +23,8 @@ import org.scalatest.time.{ Millis, Span }
 object AkkaSpec {
   val testConf: Config = ConfigFactory.parseString("""
       akka {
-        loggers = ["akka.testkit.TestEventListener"]
-        loglevel = "WARNING"
+        loggers = ["akka.testkit.SilenceAllTestEventListener"]
```

Comment:
One thing I think I noticed somewhere is that for a test with several interacting actor systems (don't know if it was multi-jvm tests or one of our remoting tests with multiple AS) it will only output logs from one of them. Could you check how it looks like for some of such tests?

Reply:
Yep that is an issue, i'll ponder that some

Reply:
I like the idea in general.